usolitaire/utils.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from .deck import Deck
+from typing import Literal
+
+def suit_color(suit: Literal["diamonds", "hearts", "clubs", "spades"]) -> Literal["red", "black"]:
+    return "red" if suit in ("diamonds", "hearts") else "black"
+
+def rank_diff(first: str, second: str) -> int:
```

Comment:
the type hinting here is incorrect, you should use the Rank and Suite defined in the other file